routing/route1.yml

```diff
@@ -0,0 +1,76 @@
+nodes:
+    - node:
```

Comment:
`-node` is unnecessary. It is implied that each `-` under `nodes:` is a node. `-` maps to a list item in python. So given...
```yaml
nodes:
    - name: A
       connected_nodes:
           - name: B
```
should map to...
```python
{
  'nodes': [
    {
      'name': 'A'
        'connected_nodes': [
          {
            'name': 'B'
          }
         # ...
        ]
      }
    # ...
   ]
}
```

Reply:
Although I agree it is not necessary, I would argue that it adds better definition and meaning to the YAML file for use within python. I am not making this change.

Reply:
I am not really suggesting it. I am saying you need to make this change as it adds an unnecessarily level of complexity given that it causes your code to be like so...
```python
for node in cfg['nodes']:
        print(node)
        graph.add_vertex(node['node']['name'])
```
vs. without it
```python
for node in cfg['codes']:
    graph.add_vertex(node['name'])
```
That is redundant

Reply:
Yeah so, I'm done making changes. This is not, nor will it ever be a dictatorship. This is my code, and I believe it should be there, I've spent 6 hours rewriting this entire WORKING program to fit your specifications. It's just stupid at this point. I'm leaving it how it is, fix it yourself later.

Reply:
Okay you are taking this personally. This isn't an indictment of you. We are talking about code. You already agreed that it was an unnecessary complexity. It adds redundant levels to the code.

Reply:
I'm not taking this personally, you're just being absolutely ridiculous about this stuff. If an outside person is looking in, it would make more sense for them to see the nodes defined individually, therefore it should stay. This is a simple project for senior design, not a production release for a major company, you need to chill man.